packages/sak-app/src/app/AppConfigResolver.tsx

```diff
@@ -53,9 +46,13 @@ const AppConfigResolver = ({ children }: OwnProps) => {
     harHentetFerdigKodeverk &&
     navAnsattState === RestApiState.SUCCESS &&
     sprakFilState === RestApiState.SUCCESS &&
-    (featureToggleState === RestApiState.NOT_STARTED || featureToggleState === RestApiState.SUCCESS);
+    !!featureToggles;
```

Comment:
useFeatureToggles() ser for meg ut til å returnere eit tomt objekt før respons er henta. Det betyr vel at `!!featureToggles` blir true før featureToggles faktisk har blitt satt? Dermed kan rendering av children potensielt starte med tomt featureToggles objekt for deretter å bli re-rendra?

Ville det vere betre om useFeatureToggles returnerte objekt med featureToggles prop som er undefined før det er henta, istadenfor array?

```typescript
const transformData = (data: FeatureToggle[]): FeatureToggles =>
  data?.reduce((acc: { [x: string]: boolean }, curr: FeatureToggle) => {
    acc[curr.key] = `${curr.value}`.toLowerCase() === 'true';
    return acc;
  }, {});

export const useFeatureToggles = (): {featureToggles: FeatureToggles | undefined} => {
  const backendUrl = window.location.pathname.includes('/ung/web') ? 'ung' : 'k9';
  const { data: featureToggles } = useQuery({
    queryKey: ['featureToggles'],
    queryFn: ({ signal }) =>
      axios.get(`/${backendUrl}/feature-toggle/toggles.json`, { signal }).then(({ data }) => data),
    select: transformData,
  });

  return {featureToggles}
};
```

Og så i her i AppConfigResolver:
```typescript
const {featureToggles} = useFeatureToggles();
```
Då ville vel erFerdig bli satt meir korrekt trur eg.